pelican/plugins/seo/tests/test_seo_report.py

```diff
@@ -75,3 +75,47 @@ def test_generate_create_report_file_and_write_output(
             args, _ = mocked_file_handle.write.call_args_list[0]
             output = args[0]
             assert "<h1>SEO report - Fake site</h1>" in output
+
+            # title
```

Comment:
Can you please create a new test for this? I prefer to have one test for the file generation and another test for the report content.

Reply:
Done. I've extracted the report generation from `test_generate_create_report_file_and_write_output()` to a separate method since it's used in two test cases now. Let me know what you think please.

Reply:
I would go for a pytest.fixture in this case (just after the import part) insteaf of a private method, called for example `seo_report`. Thus, you'll be able to call the two tests with something like this def my_test(seo_report).

Otherwise, the rest is good to me :)

Reply:
Makes sense. Changed.